tests/rptest/tests/partition_movement_test.py

```diff
@@ -127,7 +127,8 @@ def _move_and_verify(self):
         self.logger.info(
             f"new assignments for {topic}-{partition}: {assignments}")
 
-        admin.set_partition_replicas(topic, partition, assignments)
+        r = admin.set_partition_replicas(topic, partition, assignments)
+        r.raise_for_status()
```

Comment:
nothing to change in this pr, but i'm curious: should we do this by default when writing python wrappers in ducktape that use `requests` library for more precise errors?

Reply:
Yes, there are loads of places we're ignoring errors.  KafkaCliTools is a particularly bad one https://github.com/vectorizedio/redpanda/issues/2318